### DISCUSSION TITLE
Mount size

### DISCUSSION THREAD

**Discussion by macosupdate** [seaweedfs/seaweedfs#3362] [Q&A] [answered]
I have four hard drives totaling more than 20TB
But when I mount it, I only get 1.1TB.
If I store more than 1.1TB, will the size increase by itself

/dev/sda1         6.0T  189G  5.5T   4% /data2
/dev/sdc1         6.0T  160G  5.5T   3% /data3
/dev/sdd1         6.0T  128G  5.6T   3% /data4
localhost:8888:/  1.1T  664G  344G  66% /img

![](https://imgur.com/9Ugasqr)

**Comment by chrislusf** [comment, marked as answer]
Depends on how many volumes you allocated and what is the volume size limit you set.
